targets/unity-v2/source/ExampleTestProject/Assets/Testing/Editor/UUnitTestRunner.cs

```diff
@@ -27,16 +25,17 @@ private static void ClearDebugLog()
             method.Invoke(new object(), null);
         }
 
+        // From an empty scene, this will create the test scene
         [MenuItem("PlayFab/Testing/Play and Run Tests")]
         public static void TestImmediately()
         {
             ClearDebugLog();
 
-            _testers = UnityEngine.Object.FindObjectsOfType<UUnitIncrementalTestRunner>();
-            if (_testers.Length == 0)
+            var testComponents = UnityEngine.Object.FindObjectsOfType<UUnitIncrementalTestRunner>();
```

Comment:
>UUnitIncrementalTestRunner [](start = 70, length = 26)

I'm seeing an Error in Unity 2019.2 here stating "Assets\Testing\Editor\UUnitTestRunner.cs(34,71): error CS0246: The type or namespace name 'UUnitIncrementalTestRunner' could not be found (are you missing a using directive or an assembly reference?)"

[This also appears on line 37]

This file may need to be moved to the Assets/Testing/Tests/Shared/UUnit

Or PlayFab.UUnit classes may need to be in their own asmdef file to be added to the local PlayFabTestingEditor.asmdef file

Reply:
It's possible that these will be fixed with the addition of an asmdef file that I missed.
Please try again.